dotnet/samples/KernelSyntaxExamples/Example22_OpenApiPlugin_AzureKeyVault.cs

```diff
@@ -83,15 +82,15 @@ public static async Task AddSecretToAzureKeyVaultAsync(InteractiveMsalAuthentica
             });
 
         // Add arguments for required parameters, arguments for optional ones can be skipped.
-        var contextVariables = new ContextVariables();
-        contextVariables.Set("server-url", TestConfiguration.KeyVault.Endpoint);
-        contextVariables.Set("secret-name", "<secret-name>");
-        contextVariables.Set("api-version", "7.0");
-        contextVariables.Set("value", "<secret-value>");
-        contextVariables.Set("enabled", "<enabled>");
+        var arguments = new KernelFunctionArguments();
+        arguments["server-url"] = TestConfiguration.KeyVault.Endpoint;
+        arguments["secret-name"] = "<secret-name>";
+        arguments["api-version"] = "7.0";
+        arguments["value"] = "<secret-value>";
+        arguments["enabled"] = "<enabled>";
```

Comment:
Use index initializers when possible. Less verbose.

```suggestion
        var arguments = new KernelFunctionArguments 
        {
           ["server-url"] = TestConfiguration.KeyVault.Endpoint,
           ["secret-name"] = "<secret-name>",
           ["api-version"] = "7.0",
           ["value"] = "<secret-value>",
           ["enabled"] = "<enabled>",
        };
```